build.cake

```diff
@@ -131,25 +150,44 @@ Task("Test")
 	.IsDependentOn("Build")
 	.Does(() =>
 	{
-		NUnit3(UNIT_TEST_PATH);
+		NUnit3(BIN_DIR + UNIT_TEST_ASSEMBLY);
 	});
 
 //////////////////////////////////////////////////////////////////////
 // PACKAGE
 //////////////////////////////////////////////////////////////////////
 
-Task("Package")
+Task("PackageNuGet")
 	.IsDependentOn("Build")
 	.Does(() => 
 	{
-		CreateDirectory(PACKAGE_DIR);
-
-        NuGetPack(NUSPEC_PATH, new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = BIN_DIR,
-            OutputDirectory = PACKAGE_DIR
-        });
+        NuGetPack("nuget/nunit-project-loader.nuspec",
+			new NuGetPackSettings()
+			{
+				Version = nugetVersion ?? packageVersion,
+				OutputDirectory = OUTPUT_DIR,
+				Files = new [] {
+					new NuSpecContent { Source = "../LICENSE.txt" },
```

Comment:
Yes, files specified in this way are relative to the location of the nuspec file. Originally, all paths were absolute with many more constants. I wanted to make it lighter weight and simpler, but it sounds as if it's a little confusing. OTOH, I really like using just the filename for those things that are in the same directory as the choco file, as it makes it clear (at least to me) that that's where they are. 

Let's see what others think since it's purely a matter of style.